modules/openy_gc_storage/openy_gc_storage.install

```diff
@@ -75,3 +70,57 @@ function openy_gc_storage_update_8003() {
     'field.storage.node.field_gc_video_category',
   ]);
 }
+
+/**
+ * Add category field to blog post + Make event category multiple.
+ */
+function openy_gc_storage_update_8004() {
+  $config_dir = drupal_get_path('module', 'openy_gc_storage');
+  $config_dir .= '/config/install/';
+  // Update configuration.
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'core.entity_form_display.node.vy_blog_post.default',
+    'core.entity_view_display.node.vy_blog_post.default',
+    'core.entity_view_display.node.vy_blog_post.teaser',
+    'field.field.node.vy_blog_post.field_gc_video_category',
+    'field.storage.eventinstance.field_ls_category',
+    'field.storage.eventseries.field_ls_category',
```

Comment:
not enough for storage configs. You need to use a different code here to have fully supported storage updates
See https://github.com/ymcatwincities/openy/pull/2215/files#diff-b205ecc1e2f71f78ba8f3db948e7cd27R108